### PR TITLE
Add compute sendgrid example

### PR DESCRIPTION
**Pull request by shun-fan**
Add a compute engine sendgrid sample, with readme

### CODE REVIEW

**Comment by shun-fan** [comment]
@tswast  fyi since les is out today

File: compute/sendgrid/src/main/java/com/example/compute/sendgrid/SendEmailServlet.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+package com.example.compute.sendgrid;
+
+import com.sendgrid.SendGrid;
+import com.sendgrid.SendGridException;
+
+// [START example]
+public class SendEmailServlet {
+  final static String SENDGRID_API_KEY = "YOUR-SENDGRID-API-KEY";
+  final static String SENDGRID_SENDER = "YOUR-SENDGRID-SENDER";
```
Comment (phunl) [inline compute/sendgrid/src/main/java/com/example/compute/sendgrid/SendEmailServlet.java]: Is the value for sendgrid-sender an email address? Maybe something like "SENDER-EMAIL" to be consistent with "DESTINATION-EMAIL"?

Comment (lesv) [inline compute/sendgrid/src/main/java/com/example/compute/sendgrid/SendEmailServlet.java]: +1

**Comment by lesv** [comment]
LGTM w/ change of YOUR-SENDGRID-SENDER to YOUR-SENDGRID-FROM-EMAIL or YOUR-SENDGRID-SENDER-EMAIL.

For the record, I would prefer we figure out how to do these w/o editing, so that mvn appengine:deploy -DAPIKEY=blah -DSENDER_EMAIL=me@example.org

**Pull request merged**